src/utils/validate.js

```diff
@@ -0,0 +1,16 @@
+const { validationResult } = require("express-validator");
+
+const validate = validations => {
+  return async (req, res, next) => {
+    await Promise.all(validations.map(validation => validation.run(req)));
+
+    const errors = validationResult(req);
+    if (errors.isEmpty()) {
+      return next();
+    }
+
+    res.status(400).json({ errors: errors.array() });
```

Comment:
I would expect `status` to be dynamic and possible to define on the validator :) `400` is a good default value, if it's not overwritten